Faq (#415)
w3f/General-Grants-Program: grants/faq.md

#### Web3 Foundation Grants Programs'

# :interrobang: Frequently Asked Questions :interrobang:<!-- omit in toc -->

- [🧭 Global (Open & General grants)](#-global-open--general-grants)
- [🔓 Open Grants](#-open-grants)
- [💰 General Grants](#-general-grants)
- [🚚 Milestone Delivery](#-milestone-delivery)

## 🧭 Global (Open & General grants)

<details>
  <summary><b>How much can I ask for?</b></summary>

>   The Open Grants program is capped at US$ 30k for initial grants, meaning a project can only apply for 30k. If the grant is successfully completed, you can apply for a follow-up grant, which is capped at 100k. The General Grants program (https://github.com/w3f/General-Grants-Program/) is capped at 100k, although higher grants are possible in exceptional cases. However, we only accept applications for the General grants program if you have a reason for the project to remain private or you can only take fiat payments.

Reviewer: ```suggestion
  The [Open Grants](https://github.com/w3f/Open-Grants-Program) program is capped at US $30k for initial grants. If the first grant is successfully completed, you can apply for a follow-up grant, which are capped at $100k. The [General Grants](https://github.com/w3f/General-Grants-Program/) program, on the other hand, is capped at $100k for all applications, although higher grants are possible in exceptional cases. However, we only accept applications for the General Grants program if you have a reason for the project to remain private or you can only take fiat payments, or in exceptional cases if you can put together a strong case against applying for an initial $30k grant.
```

</details>

<details>
  <summary><b>How long does it take from application to decision?</b></summary>

>   Depending on the quality of the application and desirability for the ecosystem, an OPEN grant application could be approved within a week. Usually, there will be a discussion and requests for changes, additions or improvements. If no one in the committee finds the application approval-worthy, it will be closed after two weeks of inactivity. For GENERAL grants, the council convenes once a month to discuss open applications, meaning: once an editor finds your application sufficient, it'll be at most one month before it is decided upon.

Reviewer: ```suggestion
  Depending on the quality of the application and desirability for the ecosystem, an OPEN grant application could be approved within a week. Usually, there will be a discussion and requests for changes, additions or improvements. If no one in the committee finds the application approval-worthy or you don't react to our comments, it will be closed after two weeks of inactivity. For GENERAL grants, the council convenes once a month to discuss open applications, meaning: once an editor finds your application sufficiently elaborate, it'll take at most one month until a decision is made.
```

</details>

<details>
  <summary><b>Can I get an upfront payment?</b></summary>

>   No.

Reviewer: NO

</details>

<details>
  <summary><b>When do I get paid?</b></summary>

>   Payments are done for successfully submitting a milestone. By ‘successful’, we mean that our Grants team has reviewed and officially accepted your submission.

Reviewer: ```suggestion
  Payments are issued once a milestone has been successfully delivered. By ‘successful’, we mean that our Grants team has reviewed and officially accepted your submission.
```

</details>

<details>
  <summary><b>Can I reuse someone else’s open-source code?</b></summary>

>   Open source software and the web3 movement are all about collaboration. As long as you meet the code’s license, we encourage you to find, modify and contribute to already existing libraries and projects if it is of use for your project. However, we expect you to honor other people’s work and their right to attribution and your published code to adhere to the license requirements of the code you are benefitting from. Submitting code as part of a grant that violates someone else’s license will result in immediate termination of the grant. We will furthermore continue to monitor any repositories you may have submitted as part of a milestone for possible license infringements and reserve the right to terminate the grant if we find you going out of your way to hide external contributions.

Reviewer: ```suggestion
  Open source software and the Web3 movement are all about collaboration. As long as you meet the code’s license, we encourage you to find, modify and contribute to already existing libraries and projects if it is of use for your project or the community as a whole. However, we expect you to honor other people’s work and their right to attribution, and your published code to adhere to the license requirements of the code you are benefiting from. Submitting code that violates someone else’s license as part of a grant will result in immediate termination. We will furthermore continue to monitor any repositories you may have submitted as part of a milestone for possible license infringements and reserve the right to terminate the grant if we find you going out of your way to hide external contributions.
```

</details>

<details>
  <summary><b>I found one of my deliverables to be unnecessary, impossible or already done elsewhere. What do I do?</b></summary>

  Plans change. If you find parts of your original grant application to be unnecessary or you decide to pivot, but you still want to finish the project: get in touch with us. If your new plans are in line with the Web3 Foundation’s values and the council approves the amendment, you can continue your work. If your plans change significantly or you find yourself not being able to finish the grant, we can mutually agree to terminate the grant early. You are always welcome to reapply another time.
</details>

<details>
  <summary><b>I am starting a company that [...]. I want to use Polkadot/Kusama/Substrate to build a blockchain/parachain and connect [...]. Would I be eligible for a grant?</b></summary>

>   What the Web3 Foundation is mainly looking for to support are projects "[driving advancement and adoption of decentralized software protocols [and] that make it easier for developers to build useful applications using these protocols.](https://web3.foundation/grants/)" As such, we do not give grants to individual companies developing their private infrastructure. However, if part of your work is to build a library or other piece of code that could be of interest to the general Polkadot/Kusama/Substrate ecosystem and ask for funding specific to that, we are happy to look into it. Our [tech stack](https://github.com/w3f/General-Grants-Program/blob/master/grants/polkadot_stack.md) lists some applications and application-oriented grants we have given.

Reviewer: ```suggestion
  What the Web3 Foundation is mainly looking for to support are projects "[driving advancement and adoption of decentralized software protocols [and] that make it easier for developers to build useful applications using these protocols.](https://web3.foundation/grants/)" As such, we do not award grants to individual companies developing their private infrastructure. However, if part of your work is to build a library or another piece of software that could be of interest to the general Polkadot/Kusama/Substrate ecosystem and ask for funding specific to that, we are happy to look into it.
```

</details>

<details>
  <summary><b>Can I list the Web3 Foundation as a partner?</b></summary>

  No. Once the grants team has accepted your first milestone, you may display our [grants badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md) in a project-specific context, such as the repository containing the grant project work.
</details>

<details>
  <summary><b>Can you help me advertise my project?</b></summary>

>   The Web3 Foundation does not provide PR services to its grantees. However, once per month we will publish all newly signed grants on Twitter. Furthermore, if you would like our opinion on how to write about your grant, we have some general [announcement guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/announcement-guidelines.md). This document also lists an email address through which you can get in touch with our PR team, in case you have any specific questions.

Reviewer: ```suggestion
  The Web3 Foundation does not provide PR services to its grantees. However, once per month we will publish all newly signed grants on Twitter. Furthermore, if you would like our opinion on how to write about your grant, we have some general [announcement guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/announcement-guidelines.md). This document also lists an email address through which you can get in touch with our PR team in case you have specific questions.
```

</details>

<details>
  <summary><b>Can anyone apply?</b></summary>

>   Teams that have sold their own token are not eligible for a Web3 Foundation grant. Other than that, there are no restrictions.

Reviewer: ```suggestion
  Projects for which a token sale has been conducted are not eligible for a Web3 Foundation grant. Other than that, there are no restrictions.
```

</details>

<details>
  <summary><b>My application was rejected. Do you have any recommendations on where to go from here?</b></summary>

  We usually give reasons why an application was rejected. We always try to be constructive and work with you towards an application that is beneficial to all parties. If we find no common ground, please have a look at [this section in our General Grants readme](https://github.com/w3f/General-Grants-Program#rocket-alternative-funding-sources) for a list of alternative funding opportunities.
</details>

<details>
  <summary><b>One of your grantees is using my code without respecting the terms of its license.</b></summary>

  Please [reach out to us](mailto:grants@web3.foundation) asap.
</details>

<details>
  <summary><b>Something came up and I cannot finish the project in time. Can we postpone or call off the rest of my project?</b></summary>

>   The Web3 Foundation reserves the right to terminate an agreement that is behind schedule. However, we are not interested in taking away your grant for any slight hiccup. More often than not, delays are part of the journey and do not constitute a reason for concern. The best way to handle changes in your plans is to get in touch with us. If you like to prematurely end your work, we can amend your application and remove the milestones you won't be able to complete. If you decide to continue work at a later date, you can always reapply for the remaining milestones and potentially adapt them to take into account any insights you have gained in the meantime.

Reviewer: ```suggestion
  The Web3 Foundation reserves the right to terminate an agreement that is behind schedule. However, we are not interested in taking away your grant for any slight hiccup. More often than not, delays are part of the journey and do not constitute a reason for concern. The best way to handle changes in your plans is to get in touch with us. If you would like to prematurely end your work, we can amend your application and remove the milestones you won't be able to complete. If you decide to continue work at a later date, you can always reapply for the remaining milestones and potentially adapt them to take into account any insights you have gained in the meantime.
```

</details>

<details>

>   <summary><b>Why are there grant applications being accepted or rejected that were submitted after mine?</b></summary>

Reviewer: ```suggestion
  <summary><b>Why are other grant applications being accepted faster than mine?</b></summary>
```

  There are many reasons why your application might take longer than others: some applications are straightforward and simple and address an obvious issue, others require deeper understanding and discussion. If your application is highly technical or specialised, we might have to bring in an external evaluator. Sometimes, this specialised evaluator is busy with another evaluation. And sometimes, the committee is simply unsure or not quite convinced.
</details>

## 🔓 Open Grants

<details>
  <summary><b>When can I apply for a follow-up grant?</b></summary>

>   Anyone who has successfully completed a grant project (all milestones were accepted, or the previous grant was nullified in mutual agreement) can apply for a follow-up grant.

Reviewer: I would opt for the same terminology as elsewhere since we don't use `nullify` anywhere else and this might confuse people. I think the "in mutual agreement" is clear enough. Otherwise I also prefer `cancelled`

```suggestion
  Anyone who has successfully completed a grant project (i.e. all milestones were accepted, or the previous grant was terminated in mutual agreement) can apply for a follow-up grant.
```

</details>

<details>
  <summary><b>A W3F member approved my application. Does that mean it is accepted?</b></summary>

  Open Grants applications require one third of the committee to approve your pull request. Since we have many different members with different backgrounds and specializations, it is possible that the committee disagrees and your application gets rejected even though one or two members approved it.
  The application is accepted once the pull request is merged.
</details>

> 
> <details>
>   <summary><b>A W3F member approved my application. Does that mean it is accepted?</b></summary>
> 
>   Open Grants applications require one third of the committee to approve your pull request. Since we have many different members with different backgrounds and specializations, it is possible that committee members disagree on an issue and your application gets rejected even though one or two members approved it.
>   The application is accepted once the pull request is merged.
> </details>

Reviewer: Duplicate

```suggestion

```

## 💰 General Grants

<details>
  <summary><b>How do I apply for a General Grant? I see a Google form, a Github repository, requests for a document…</b></summary>

>   _Every_ General Grant application requires a [form submission](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform). If you like the application to be completely private, you will have to provide all the details about your plans, team, milestones, financials, etc. in a document that you can attach to the form. Our [template](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant_application_template.md) shows what kind of information we are looking for.

Reviewer: ```suggestion
  _Every_ General Grant application requires a [form submission](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform). If you would like the application to be completely private, you will have to provide all the details about your plans, team, milestones, financials, etc. in a document that you can attach to the form. Our [template](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant_application_template.md) shows what kind of information we are looking for.
```

  For (partially) public applications, you are free to submit this information as part of the pull request instead or in addition. What information goes where is up to you, as long as we get all the required information.
</details>

## 🚚 Milestone Delivery

<details>
  <summary><b>How do I submit a milestone?</b></summary>

>   For details, please refer to the milestone delivery guidelines for your grants program (Open Grants, General Grants). Generally speaking, the most important part of a delivery is a list of deliverables with links to their implementation/realisation (_ideally_ linking to a specific commit, so you can continue working on your repository without messing up your delivery and complicating our evaluation) and any additional notes you might have. The list of deliverables for each of your milestones should be defined in your grant agreement.

Reviewer: ```suggestion
  For details, please refer to the milestone delivery guidelines for the respective grants program. Generally speaking, the most important part of a delivery is a list of **the same deliverables listed in the application** with links to their implementation/realisation (ideally pointing to a specific commit or tag, so you can continue working on your repository without messing up your delivery and complicating our evaluation) and any additional notes you might have.
```

</details>

<details>
  <summary><b>Can I submit two or more milestones at once?</b></summary>

>   You can. However, we strongly encourage you to submit your work in increments (milestones), so that you can be sure we didn’t misunderstand (an aspect of) your application, and you didn't make changes to your plan or delivery that would have required a reevaluation of the application.

Reviewer: ```suggestion
  You can. However, we strongly encourage you to submit your work in increments of one milestone each, so that you can be sure we didn’t misunderstand an aspect of your application and in order to minimise the impact of potential changes to your deliveries that we might request.
```

</details>

<details>
  <summary><b>Can I add a badge to my repo once I’ve completed a milestone?</b></summary>

  Yes, after your first milestone has been merged and as long as you follow the [badge guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md).
</details>

<details>

>   <summary><b>Why are there milestones being accepted or discussed that were submitted after mine?</b></summary>

Reviewer: ```suggestion
  <summary><b>Why are other milestones being accepted or discussed faster than mine?</b></summary>
```

>   While we try to process deliveries chronologically, some milestones (_as well as_ applications) do not get processed quite as fast as others. One obvious reason is the complexity of the delivery and its evaluation. Other times, your submission might require internal discussion or delegation. In any case, if you have any question on the processing of your delivery, you can reach out to us via email or Github.

Reviewer: ```suggestion
  While we try to process deliveries chronologically, some milestones aren't processed quite as fast as others. One obvious reason is the complexity of the delivery and its evaluation. Other times, your submission might require internal discussion or delegation. In any case, if you have any question on the processing of your delivery, you can reach out to us via email or Github.
```
